python/ray/tune/tests/test_ray_trial_executor.py

```diff
@@ -16,11 +17,12 @@
 from ray.tune.resources import Resources
 from ray.cluster_utils import Cluster
 from ray.tune.utils.placement_groups import PlacementGroupFactory
+from unittest.mock import patch
 
 
 class TrialExecutorInsufficientResourcesTest(unittest.TestCase):
     def setUp(self):
-        os.environ["TUNE_WARN_INSUFFICENT_RESOURCE_THRESHOLD_S"] = "1"
+        os.environ["TUNE_WARN_INSUFFICENT_RESOURCE_THRESHOLD_S"] = "0"
```

Comment:
what is this doing?

Reply:
oh this is just to make the threshold literally 0s. So that we can run the test faster, i.e. see the warning in less time.